drivers/adc/adc_ads1x1x.c

```diff
@@ -544,6 +613,70 @@ static void ads1x1x_acquisition_thread(const struct device *dev)
 	}
 }
 
+#ifdef CONFIG_ADC_ADS1X1X_TRIGGER
+static void adc_ads1x1x_work_fn(struct k_work *work)
+{
+	struct ads1x1x_data *data;
+	const struct device *dev;
+
+	data = CONTAINER_OF(work, struct ads1x1x_data, work);
+	dev = data->dev;
+
+	ads1x1x_adc_perform_read(dev);
+}
+
+static void adc_ads1x1x_conv_ready_cb(const struct device *gpio_dev,
```

Comment:
nit: naming of function doesn't seem to be consistent with the others, maybe change name to `ads1x1x_conv_ready_cb`

Reply:
done